src/python/pants/backend/codegen/utils.py

```diff
@@ -0,0 +1,62 @@
+# Copyright 2022 Pants project contributors (see CONTRIBUTORS.md).
```

Comment:
Place seems fine. Maybe rename from "utils" to something more related to its function? For example, "dep_utils" since related to dependencies. (Although probably a better naming than "dep_utils.")